File: Model/Employee.cs
namespace Model;

public class Employee
{
    public string FullName { get; set; } = "";
    public string Email { get; set; } = "";
    public string PhoneNumber { get; set; } = "";
    public DateTime DateOfBirth { get; set; }
    public decimal MonthlySalary { get; set; }
    public long Id { get; set; }
    public List<WorkTask> WorkTasks { get; set; }=new List<WorkTask>();
}

File: ClientApp/src/app/models/employee.ts
import { WorkTask } from "./workTask";

export class Employee
{
    id:number=0;
    fullName:string="";
    email:string="";
    phoneNumber:string="";
    dateOfBirth:Date=new Date();
    monthlySalary:number=0;
    workTasks:Array<WorkTask>=new Array<WorkTask>();
}
 